windows/tools/tag_scanner_window.py

```diff
@@ -32,7 +32,7 @@
     # leaving this here just in case it somehow works though.
     TEXT_EDITOR_NAME = "TextEdit"
 elif platform == "linux":

```

Comment:
Might not work for certain Linux versions. Needs to be
```suggestion
elif "linux" in platform:
```